### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by lowks** [mlen108/pipin#7]
I did `python setup.py develop` then `python setup.py test`, this is the result ? 

```
self = <setuptools.tests.test_resources.NamespaceTests testMethod=test_two_levels_deep>

    def test_two_levels_deep(self):
        """
            Test nested namespace packages
            Create namespace packages in the following tree :
                site-packages-1/pkg1/pkg2
                site-packages-2/pkg1/pkg2
            Check both are in the _namespace_packages dict and that their __path__
            is correct
            """
        sys.path.append(os.path.join(self._tmpdir, "site-pkgs2"))
        os.makedirs(os.path.join(self._tmpdir, "site-pkgs", "pkg1", "pkg2"))
        os.makedirs(os.path.join(self._tmpdir, "site-pkgs2", "pkg1", "pkg2"))
        ns_str = "__import__('pkg_resources').declare_namespace(__name__)\n"
        for site in ["site-pkgs", "site-pkgs2"]:
            pkg1_init = open(os.path.join(self._tmpdir, site,
                             "pkg1", "__init__.py"), "w")
            pkg1_init.write(ns_str)
            pkg1_init.close()
            pkg2_init = open(os.path.join(self._tmpdir, site,
                             "pkg1", "pkg2", "__init__.py"), "w")
            pkg2_init.write(ns_str)
            pkg2_init.close()
        import pkg1
>       self._assertIn("pkg1", pkg_resources._namespace_packages.keys())

pipin/lib/python2.7/site-packages/setuptools/tests/test_resources.py:603: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <setuptools.tests.test_resources.NamespaceTests testMethod=test_two_levels_deep>, member = 'pkg1', container = []

    def _assertIn(self, member, container):
        """ assertIn and assertTrue does not exist in Python2.3"""
        if member not in container:
            standardMsg = '%s not found in %s' % (safe_repr(member),
                                                  safe_repr(container))
>           self.fail(self._formatMessage(msg, standardMsg))
E           NameError: global name 'msg' is not defined
```

**Comment by mlen108** [comment]
Thanks, I will investigate this. FYI I never did `python setup.py develop` - it's always `install`.

**Comment by mlen108** [comment]
It's probably failing because tests relay on `subprocess.Popen` results for `pipin`. I'm going to change it in a way it actually tests the codebase and not the above command.

**Comment by mlen108** [comment]
Fixed now, and released as 0.2.2.
